### PR TITLE
add new solidifier hatches for industrial multi machine

### PR DESCRIPTION
**Pull request by Quetz4l**
A copy of https://github.com/GTNewHorizons/GTplusplus/pull/677 , because there are too many changes  from last pr

### CODE REVIEW

**Comment by chochem** [comment]
I thought the plan was to go to GT?

**Comment by Quetz4l** [comment]
It only for 1 gt++ machine, so I don't think it should be put in gregtech. The whole pr idea is the same as the last one. If you want to make something universal for everyone, you need to discuss how to do what etc. Also need someone more skillful than me

**Comment by Pelotrio** [comment]
Not really sure if this is necessary and I am not too much of a fan of this implementation. Also does the 50% bonus speed still exist?

**Comment by Quetz4l** [comment]
no bonuses this time

File: src/main/java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_Hatch_Solidifier.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    public GT_MetaTileEntity_Hatch_Solidifier(int aID, String aName, String aNameRegional, int aTier) {
+        super(aID, aName, aNameRegional, aTier);
+    }
+
+    @Override
+    public String[] getDescription() {
+        return new String[] {
+                "Fluid Input with Mold for " + EnumChatFormatting.YELLOW
+                        + "Large Processing Array"
```
Comment (Laiff) [inline src/main/java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_Hatch_Solidifier.java]: `typo` Factory

**Review by Laiff — APPROVED** [review]

**Pull request merged**